datafusion/sqllogictest/test_files/insert_to_external.slt

```diff
@@ -362,8 +352,7 @@ statement ok
 CREATE EXTERNAL TABLE
 table_without_values(field1 BIGINT NULL, field2 BIGINT NULL)
 STORED AS parquet
-LOCATION 'test_files/scratch/insert_to_external/external_parquet_table_q2'
-OPTIONS (create_local_path 'true');
+LOCATION 'test_files/scratch/insert_to_external/external_parquet_table_q2/';
```

Comment:
As these paths don't exist anymore, we need to use the trailing `/` to indicate that we intend for a directory, as opposed to a file